tests/python/pytorch/graphbolt/test_csc_sampling_graph.py

```diff
@@ -212,6 +212,19 @@ def test_node_type_offset_wrong_legnth(node_type_offset):
         )
 
 
+def check_tensors_on_the_same_shared_memory(t: torch.Tensor, t2: torch.Tensor):
+    """Check if two tensors are on the same shared memory.
+
+    Cannot use `.data_ptr()` here because the memory created by different
+    SharedMemory objects have different pointers.
+    """
+    old_t1 = t.clone()
+    v = torch.randint_like(t, 100)
+    t[:] = v
+    assert torch.equal(t, t2)
```

Comment:
How does the code here checks two tensors are on the same shared memory?
Not intuitive to me, comment required.

Reply:
It copies a random tensor to the first tensor to see whether the second tensor also has the same random value. Will add a commet.